webplugin/js/app/media/media-service.js

```diff
@@ -45,13 +45,13 @@ Kommunicate.mediaService = {
     voiceOutputIncomingMessage: function (message) {
         // get appoptions
         var appOptions = KommunicateUtils.getDataFromKmSession("appOptions") || applozic._globals;
-
+        var userOverrideVoiceOutput = KommunicateUtils.getItemFromLocalStorage("USER_OVERRIDE").VOICE_OUTPUT_ENABLED;
```

Comment:
is USER_OVERRIDE guaranteed to be always present? if not then this will give undefined error

Reply:
dependency on local storage is removed now